pkg/server/handler/v1alpha1/webhook.go

```diff
@@ -60,34 +66,91 @@ func HandleWebhook(ctx context.Context, tenant, eventType, integration string) (
 	}
 
 	if request.Header.Get(bitbucket.EventTypeHeader) != "" {
-		in, err := getIntegration(common.TenantNamespace(tenant), integration)
-		if err != nil {
-			return newWebhookResponse(err.Error()), err
+		hookEvent := request.Header.Get(bitbucket.HookEventHeader)
+		tenants := make([]string, 0)
+		var terr error
+		if hookEvent == "true" {
+			serverAddress := strings.Trim(request.Header.Get(bitbucket.ServerAddressHeader), "/")
+			hookData := bitbucket.ParseHookEvent(request)
+
+			// Trigger workflows for all tenants
```

Comment:
Why?

Reply:
The hook configured in the bitbucket server can not know which tenant will be triggered.